zerver/lib/bugdown/__init__.py

```diff
@@ -1163,11 +1154,14 @@ def run(self, root: Element) -> None:
                 continue
 
             if extracted_data:
-                if youtube is not None:
+                if is_youtube_url(url):
+                    # We use a fix sized thumbnail for every YouTube preview. So,
+                    # thumbnail resolution is always converted to `default.jpg`.
+                    self.youtube_image(extracted_data)
```

Comment:
`fix_youtube_thumbnail` or something may be more appropriate than just `youtube_image`